external_parser/unit_tests/test_files/test_outputs/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
nit (don't care a lot): could possibly be migrated to root gitignore file

Reply:
yeah tried that but afaik git doesn't let you add a directory if it is empty so just added a gitignore file to it